### ISSUE TITLE
On Xenial updating to latest modsecurity2 causes error

### ISSUE THREAD

**Issue by gquipster** [oerdnj/deb.sury.org#714]
# /etc/init.d/apache2 restart
 * Restarting Apache httpd web server apache2                                                                                                                                                                                                                           [fail]
 * The apache2 configtest failed.
Output of config test was:
apache2: Syntax error on line 147 of /etc/apache2/apache2.conf: Syntax error on line 12 of /etc/apache2/mods-enabled/security2.conf: Could not open configuration file /usr/share/modsecurity-crs/owasp-crs.load: No such file or directory

Had to install Zesty version of modsecurity-crs to resolve: https://packages.ubuntu.com/zesty/modsecurity-crs

**Comment by gquipster** [comment]
Also affects trusty

**Comment by alexiacobws** [comment]
Confirmed on Trusty:

`apt-cache policy libapache2-mod-security2 modsecurity-crs
libapache2-mod-security2:
  Installed: (none)
  Candidate: 2.9.2-1+ubuntu14.04.1+deb.sury.org+2
  Version table:
     2.9.2-1+ubuntu14.04.1+deb.sury.org+2 0
        500 http://ppa.launchpad.net/ondrej/apache2/ubuntu/ trusty/main amd64 Packages
     2.7.7-2 0
        500 http://gb.archive.ubuntu.com/ubuntu/ trusty/universe amd64 Packages
modsecurity-crs:
  Installed: (none)
  Candidate: 2.2.8-1
  Version table:
     2.2.8-1 0
        500 http://gb.archive.ubuntu.com/ubuntu/ trusty/universe amd64 Packages
`
Installed with:
`apt-get install libapache2-mod-security2 modsecurity-crs                                                                                                                                                        
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  libapache2-modsecurity
Suggested packages:
  lua geoip-database-contrib
The following NEW packages will be installed
  libapache2-mod-security2 libapache2-modsecurity modsecurity-crs
0 to upgrade, 3 to newly install, 0 to remove and 148 not to upgrade.
Need to get 0 B/638 kB of archives.
After this operation, 3,910 kB of additional disk space will be used.
Do you want to continue? [Y/n] 
WARNING: The following packages cannot be authenticated!
  libapache2-mod-security2 libapache2-modsecurity
Install these packages without verification? [y/N] y
Selecting previously unselected package libapache2-mod-security2.
(Reading database ... 84751 files and directories currently installed.)
Preparing to unpack .../libapache2-mod-security2_2.9.2-1+ubuntu14.04.1+deb.sury.org+2_amd64.deb ...
Unpacking libapache2-mod-security2 (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
Selecting previously unselected package libapache2-modsecurity.
Preparing to unpack .../libapache2-modsecurity_2.9.2-1+ubuntu14.04.1+deb.sury.org+2_all.deb ...
Unpacking libapache2-modsecurity (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
Selecting previously unselected package modsecurity-crs.
Preparing to unpack .../modsecurity-crs_2.2.8-1_all.deb ...
Unpacking modsecurity-crs (2.2.8-1) ...
Setting up libapache2-mod-security2 (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
apache2_invoke: Enable module security2
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: apache2: Syntax error on line 157 of /etc/apache2/apache2.conf: Syntax error on line 12 of /etc/apache2/mods-enabled/security2.conf: Could not open configuration file /usr/share/modsecurity-crs/owasp-crs.load: No suc
h file or directory
Setting up libapache2-modsecurity (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
Setting up modsecurity-crs (2.2.8-1) ...
`

**Comment by alexiacobws** [comment]
Confirmed on Trusty

> Setting up libapache2-mod-security2 (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
apache2_invoke: Enable module security2
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: apache2: Syntax error on line 157 of /etc/apache2/apache2.conf: Syntax error on line 12 of /etc/apache2/mods-enabled/security2.conf: Could not open configuration file /usr/share/modsecurity-crs/owasp-crs.load: No such file or directory
Setting up libapache2-modsecurity (2.9.2-1+ubuntu14.04.1+deb.sury.org+2) ...
Setting up modsecurity-crs (2.2.8-1) ...

**Comment by jaydrogers** [comment]
I just had this same exact error happen to me on 14.04.5. The error I received was:
```
apache2: Syntax error on line 147 of /etc/apache2/apache2.conf: Syntax error on line 12 of /etc/apache2/mods-enabled/security2.conf: Could not open configuration file /usr/share/modsecurity-crs/owasp-crs.load: No such file or directory
```

The only way to get the webserver to start was to remove the config file that was causing it to break:
```
rm /etc/apache2/mods-enabled/security2.load
```

Does anyone know of a permanent solution for this?

**Comment by gquipster** [comment]
I resolved this by downloading the DEB file for the newest version of modsecurity-crs from the Zesty archive library:

https://packages.ubuntu.com/zesty/all/modsecurity-crs/download

and installing it manually.

The security2.load file then loaded without issue

**Comment by jaydrogers** [comment]
Awesome, thanks @gquipster!

For anyone landing here from a Google search, I ran these commands thanks to @gquipster:

```
wget http://mirrors.kernel.org/ubuntu/pool/universe/m/modsecurity-crs/modsecurity-crs_3.0.0-3_all.deb

sudo dpkg -i modsecurity-crs_3.0.0-3_all.deb
```

I was able to confirm the configuration was okay after install by running `sudo apachectl configtest`. After that I restarted the service with `sudo service apache2 restart` and everything came up okay.

**Comment by oerdnj** [comment]
I'll fix this as part of #733

**Comment by oerdnj** [comment]
Uploaded, so this and #733 should be fixed.
